### ISSUE TITLE
npm shrinkwrap bug affecting package.json

### ISSUE THREAD

**Issue by JordanBelford** [jcoglan/node-csprng#2]
According to the [npm semver docs](https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-), an empty string "" is supposed to be treated the same as "*" or ">=0.0.0", but there is a bug with npm shrinkwrap when an empty string is used. (see https://github.com/npm/npm/issues/11493)

There is no fix for the issue on the horizon, so it would help me and anyone else facing this issue if the package.json is simply updated here to use "*" instead of "" for the sequin dependency.

**Comment by JordanBelford** [comment]
For reference, when running 'npm shrinkwrap',  the error looks like

```
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "shrinkwrap"
npm ERR! node v6.6.0
npm ERR! npm  v3.10.3

npm ERR! Problems were encountered
npm ERR! Please correct and try again.
npm ERR! invalid: have sequin@0.1.0 (expected: ) /Users/jordan.belford/zayo/tranzact3/node_modules/csprng/node_modules/sequin
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jordan.belford/zayo/tranzact3/npm-debug.log
```

**Comment by tkambler** [comment]
Ditto. @JordanBelford How are you currently handling this?

**Comment by tkambler** [comment]
FYI @jcoglan I'm seeing the same error show up for `tunnel-agent` - both dependencies of your `faye` module.

**Comment by tkambler** [comment]
Updating faye resolved the issue. Sorry for the multiple posts.

**Comment by jcoglan** [comment]
Apologies for the delay, I've just published 0.1.2 which should fix this issue.
